### DISCUSSION TITLE
Install specific version of strimzi via https://strimzi.io/install/

### DISCUSSION THREAD

**Discussion by mossad-zika** [strimzi/strimzi-kafka-operator#5057] [Q&A]
Currently I see only the option with latest strimzi

`kubectl create -f 'https://strimzi.io/install/latest?namespace=kafka' -n kafka`

I am wondering is it possible to specify the version?

Thx in advance

**Comment by scholzj** [comment]
No, I'm afraid this works only for the latest version. For older versions, you have to download it from the website or GitHub release page.

**Comment by mateuszkwiatkowski** [comment]
I was trying to do the same but it didn't work. It would be a nice user-friendly feature. :-)
